qtp_job_output_folder/tests/test_summary.py

```diff
@@ -61,36 +69,58 @@ def test_summary(self):
         # asserting content of html
         res = self.qclient.get("/qiita_db/artifacts/%s/" % aid)
         # cleaning artifact files, to avoid errors
-        [self._clean_up_files.extend([ff['filepath']])
-         for f in res['files'].values() for ff in f]
-        html_fp = res['files']['html_summary'][0]['filepath']
+        [
+            self._clean_up_files.extend([ff["filepath"]])
+            for f in res["files"].values()
+            for ff in f
```

Comment:
my intuition says that shorter variable names are inner elements. Here `f` is a list and `ff` a file-dict.
With three lines, we have more space, therefore I suggest to change `f` into `files` and `ff` into `file`